pkg/testutil/nerdtest/requirements.go

```diff
@@ -171,6 +171,20 @@ var CGroup = &test.Requirement{
 	},
 }
 
+var CgroupsAccessible = &test.Requirement{
+	Check: func(data test.Data, helpers test.Helpers) (ret bool, mess string) {
+		isRootLess := getTarget() == targetNerdctl && rootlessutil.IsRootless()
+		if isRootLess {
+			stdout := helpers.Capture("info", "--format", "{{ json . }}")
+			var dinf dockercompat.Info
+			err := json.Unmarshal([]byte(stdout), &dinf)
+			assert.NilError(helpers.T(), err, "failed to parse docker info")
+			return dinf.CgroupVersion == "2", "we are rootless, and cgroup version is not 2"
```

Comment:
You may also want to check that CgroupDriver == "systemd"

Reply:
Ok, will have a look.
Travelling currently - will be slow to answer.

Reply:
Ok, amended.